stf.core/scripts/stf/java_properties_check.pl

```diff
@@ -0,0 +1,80 @@
+#!/usr/bin/perl
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# java_properties_check.pl
+#
+# Display details about a Java.
+#
+# perl java_properties_check.pl
+# perl java_properties_check.pl -p JAVA_HOME
+# - Gets java propertiesfor the Java at the environment variable JAVA_HOME
+
+# perl java_properties_check.pl -p PATH
+# - Gets java propertiesfor the Java on the PATH
+#
+# perl java_properties_check.pl -p <path>
+# - Gets java propertiesfor the Java specified.  Assumes the directory provided in the one above bin/java
```

Comment:
Could we fix this typo? "propertiesfor" -> "properties for".

Reply:
Or even better if we can remove the commented out lines. They seem redundant as we seem to have the same block of code from line 70-80 on the same file.